tests/installation/installation_overview.pm

```diff
@@ -62,6 +62,30 @@ sub ensure_ssh_unblocked {
     }
 }
 
+sub check_default_target {
```

Comment:
what about check_default_systemd_target?

Reply:
I found that it is named like that googling a bit, I thought the same, but it is kind of `check_default.target_file`, so I prefer this name, it is more simple/generic, previous code had it the same in its comments.